docs/faq/frequently_asked_questions.rst

```diff
@@ -0,0 +1,56 @@
+
+#. How do I get telemetry from my motors? How do I get Flight Controller logs?
+    * Please refer to our documentation about :ref:`Exporting Logs <ifci_log>`.
+    
+#. What causes electromagnetic interference (EMI) between the motor and the integrated ESC? Is electrical noise an issue when it comes to long communication wires? 
+    * Electromagnetic interference (EMI) can be radiated, coupled, or conducted at different frequencies while the motor is spinning. Electrical noise that is coupled to the motor’s body can be conducted to conduits carrying communications cables, such as the vehicle’s frame or drone arm. Longer communication wires can lead to more noise. To learn more about EMI and how to mitigate electrical noise, please refer to the :ref:`Understanding and Debugging Communication Errors from Motor Noise <motor_noise_debugging>` documentation.
+
+#. What flight controllers do you support? What firmware can I use (Ardupilot, PX4, etc.) ?
+    * Vertiq modules have been tested primarily with flight controllers using :ref:`PX4 <ifci_px4_flight_controller>` and :ref:`Ardupilot <dronecan_fc_tutorial>` firmware, and our documentation is focused on those types of flight controllers and their accompanying hardware. Other types of flight controllers may be used if they support protocols that can communicate with a Vertiq module, refer to your flight controllers documentation.
+
+#. I’m using voltage/velocity input commands through the Control Center. Why is my motor spinning counterclockwise? What determines which direction my motor spins? Why is it spinning the wrong way?
+    * Voltage and Velocity inputs do not use the module's motor direction configuration, and will always result in a counter-clockwise rotation given a positive voltage/velocity input, while a negative voltage/velocity input will always result in a clockwise rotation. Please refer to our documentation about :ref:`Velocity and Voltage Based Control Mechanisms <manual_velocity_control_mechanisms>` for more information. 
```

Comment:
, and will always result in a counter-clockwise rotation given a positive voltage/velocity input. A negative voltage/velocity...